src/cmd/tools.go

```diff
@@ -73,7 +73,7 @@ var readCredsCmd = &cobra.Command{
 		// Continue loading state data if it is valid
 		config.InitState(state)
 
-		fmt.Println(config.GetSecret(config.StateGitPush))
+		fmt.Println(state.GitServerInfo.GitPushPassword)
```

Comment:
Doesnt have to be in this PR overall, but would be nice to state (maybe via`message` so it's on stderr) that this is only the Git Server Push Password, not every credential.

Reply:
In the same vein, changing the command name might be useful `get-admin-password` is pretty generic and unclear unless your read the description in a help menu.